Assets/PatchKit Patcher/Scripts/Debug/DebugMenu.cs

```diff
@@ -0,0 +1,242 @@
+        {
+#if UNITY_EDITOR
+            OpenPopup("Access to Launcher in the editor is not possible");
+#else
+            string logDirectoryPath = Patcher.Instance.Data.Value.LockFilePath.Replace(".lock","");
+            Open(logDirectoryPath);
+#endif
+        }
+
+        private void OpenLauncherLogFile()
+        {
+#if UNITY_EDITOR
+            OpenPopup("Access to Launcher in the editor is not possible");
+#else
+            string logPath = Patcher.Instance.Data.Value.LockFilePath.Replace(".lock","launcher-log.txt");
+            Open(logPath);
+#endif
+        }
+
+        private void OpenPatcherLogFileLocation()
+        {
+#if UNITY_EDITOR
+            var logDirectoryPath = Application.consoleLogPath.Replace("Editor.log", "");
+#else
+#if UNITY_STANDALONE_WIN
+                var logDirectoryPath = string.Format("{0}",
+                    Application.persistentDataPath);
+#elif UNITY_STANDALONE_LINUX
+                var logDirectoryPath = string.Format("~/.config/unity3d/{0}/{1}",
+                    Application.companyName,
+                    Application.productName);
+#elif UNITY_STANDALONE_OSX
+    #if UNITY_2019_1_OR_NEWER
+                var logDirectoryPath = string.Format("~/Library/Logs/{0}/{1}",
+                    Application.companyName,
+                    Application.productName);
+    #else
+                var logDirectoryPath = "~/Library/Logs/Unity";
+    #endif
+#endif
+#endif
+            OpenFile(logDirectoryPath);
+        }
+
+        private void OpenPatcherLogFile()
+        {
+#if UNITY_EDITOR
+            var logPath = Application.consoleLogPath;
+#else
+#if UNITY_STANDALONE_WIN
+#if UNITY_2019_1_OR_NEWER
+                var logPath = string.Format("{0}/Player.log",
+                    Application.persistentDataPath);
+#else
+                    var logPath = string.Format("{0}/output_log.txt",
+                    Application.persistentDataPath);
+#endif
+#elif UNITY_STANDALONE_LINUX
+                var logPath = string.Format("~/.config/unity3d/{0}/{1}/Player.log",
+                    Application.companyName,
+                    Application.productName);
+#elif UNITY_STANDALONE_OSX
+#if UNITY_2019_1_OR_NEWER
+                var logPath = string.Format("~/Library/Logs/{0}/{1}/Player.log",
+                    Application.companyName,
+                    Application.productName);
+               
+#else
+                var logPath = "~/Library/Logs/Unity/Player.log";
+#endif
+#endif
+#endif
+            OpenFile(logPath);
+        }
+
+        void DrawPopup(int id)
+        {
+            GUILayout.Label(_popupMessage);
+            if (GUILayout.Button("OK"))
+            {
+                ClosePopup();
+            }
+        }
+
+        void Close()
+        {
+            _showDebugMenu = false;
+            _graphicRaycaster.enabled = true;
+        }
+
+        void ClosePopup()
+        {
+            _showPopup = false;
+            _graphicRaycaster.enabled = true;
```

Comment:
`_graphicRaycaster.enabled` should be managed only by `Open` and `Close` methods.

Reply:
done